corehq/apps/style/templates/style/base.html

```diff
@@ -276,7 +276,7 @@
                                     </div>
                                 {% endfor %}
                             {% endif %}
-                            <div id="message-alerts" data-bind="foreach: alerts">
+                            <div id="message-alerts" class="ko-template" data-bind="foreach: alerts">
```

Comment:
Yeah, and the `this.removeClass('ko-template');` is specifically in the `$.fn.koApplyBindings` function, so at least the way it is pretty tied to the knockout.